services/auth-gateway/src/auth/service-authenticator.ts

```
@@ -0,0 +1,181 @@
+   */
+  private isValidSPIFFEID(spiffeId: string): boolean {
+    const spiffePattern = /^spiffe:\/\/mesh\.summit\.internal\/ns\/[a-z0-9-]+\/sa\/[a-z0-9-]+$/;
+    return spiffePattern.test(spiffeId);
+  }
+
+  /**
+   * Parse SPIFFE ID to extract service information
+   */
+  private parseSPIFFEID(spiffeId: string): { serviceName: string; namespace: string } {
+    // spiffe://mesh.summit.internal/ns/<namespace>/sa/<service-account>
+    const match = spiffeId.match(/\/ns\/([^\/]+)\/sa\/([^\/]+)/);
```

ESLint

[error] Unnecessary escape character: \/.

(no-useless-escape)